src/data/Nav.js

```diff
@@ -1,4 +1,8 @@
 export const items = [
+  {
+    name: "HOME",
+    link: "home",
```

Comment:
try clicking on the home tab, it probably won't work.
Hint: the home route for a website is just the root